tests/models/test_models.py

```diff
@@ -1112,6 +1112,18 @@ def test_scanvi(save_path):
     scanvi_model.train(1)
 
 
+def test_linear_classifier_scanvi():
+    adata = synthetic_iid()
+    SCANVI.setup_anndata(
+        adata,
+        "labels",
+        "label_0",
+        batch_key="batch",
+    )
+    model = SCANVI(adata, linear_classifier=True)
+    model.train(max_epochs=1)
+
```

Comment:
let's assert that the classifier nn module has the right structure in this test as well

Reply:
done